sw/device/examples/BUILD

```diff
@@ -0,0 +1,19 @@
+package(default_visibility = ["//visibility:public"])
+
+load("//rules:opentitan.bzl", "OPENTITAN_CPU")
+
+cc_library(
+    name = "demos",
+    srcs = ["demos.c"],
+    hdrs = ["demos.h"],
+    deps = [
+        "//sw/device/lib/arch:device",
+        "//sw/device/lib/dif:gpio",
+        "//sw/device/lib/dif:spi_device",
+        "//sw/device/lib/dif:uart",
+        "//sw/device/lib/runtime:hart",
+        "//sw/device/lib/runtime:log",
+        "//sw/device/lib/testing",
+    ],
+    #target_compatible_with = [OPENTITAN_CPU],
```

Comment:
Is this needed?  I'd suspect that `demos.c` doesn't do anything riscv specific, but that many of the deps do.  I'd guess this is inherited from the deps.